src/onboarding/components/LogoWithCubo.scss

```diff
@@ -0,0 +1,15 @@
+.logo-with-cubo {
+  display: flex;
+
+  > .cubo-logo {
+    font-size: $cf-text-base-4;
+    padding-right: $cf-space-2xs;
+    color: #fff;
+    line-height: 1.5;
+  }
+  > .influx-cloud-logo {
+    width: 150px;
+    height: 50px;
+    display: flex;
+  }
+}
```

Comment:
Rather than creating a new file here, let's just add this css to [`LoginPage.scss`.](https://github.com/influxdata/ui/blob/master/src/onboarding/containers/LoginPage.scss)

If this is meant to be part of a contained package that can be used on any arbitrary page, then we can keep it, but the scss file should probably be imported in the `LogoWithCubo.tsx` file so that someone using it doesn't have to know to import the component and the stylehseet